tests/MongoDB.Analyzer.Tests.Common.TestCases/Linq/LinqQuery.cs

```diff
@@ -0,0 +1,128 @@
+        public void Query_Syntax_7()
+        {
+            var query = from item in GetMongoQueryable()
+                        where (item.Age == 22 || item.Age == 25)
+                        where (item.LastName == "Doe" || item.Name == "John")
+                        select new { item.Age, item.Address };
+        }
+
+        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"Age\" : 22 }, { \"Age\" : 25 }] } }, { \"$match\" : { \"$or\" : [{ \"LastName\" : \"Doe\" }, { \"Name\" : \"John\" }] } }, { \"$match\" : { \"Address\" : /Drive$/s } }, { \"$project\" : { \"Age\" : \"$Age\", \"Address\" : \"$Address\", \"_id\" : 0 } }])")]
+        public void Query_Syntax_8()
+        {
+            var query = from item in GetMongoQueryable()
+                        where (item.Age == 22 || item.Age == 25)
+                        where (item.LastName == "Doe" || item.Name == "John")
+                        where item.Address.EndsWith("Drive")
+                        select new { item.Age, item.Address };
+        }
+
+        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"Age\" : 22 }, { \"Age\" : 25 }] } }, { \"$match\" : { \"$or\" : [{ \"LastName\" : \"Doe\" }, { \"Name\" : \"John\" }] } }, { \"$match\" : { \"Address\" : /^Drive/s } }, { \"$project\" : { \"Age\" : \"$Age\", \"Address\" : \"$Address\", \"_id\" : 0 } }])")]
+        public void Query_Syntax_9()
+        {
+            var query = from item in GetMongoQueryable()
+                        where (item.Age == 22 || item.Age == 25)
+                        where (item.LastName == "Doe" || item.Name == "John")
+                        where item.Address.StartsWith("Drive")
+                        select new { item.Age, item.Address };
+        }
+
+        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"Age\" : 22 }, { \"Age\" : 25 }] } }, { \"$match\" : { \"$or\" : [{ \"LastName\" : \"Doe\" }, { \"Name\" : \"John\" }] } }, { \"$match\" : { \"Address\" : /^Drive/s } }, { \"$project\" : { \"Age\" : \"$Age\", \"Address\" : \"$Address\", \"Name\" : \"$Name\", \"_id\" : 0 } }])")]
+        public void Query_Syntax_10()
+        {
+            var query = from item in GetMongoQueryable()
+                        where (item.Age == 22 || item.Age == 25)
+                        where (item.LastName == "Doe" || item.Name == "John")
+                        where item.Address.StartsWith("Drive")
+                        select new { item.Age, item.Address, item.Name };
+        }
+
+        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"Age\" : 22 }, { \"Age\" : 25 }] } }, { \"$match\" : { \"$or\" : [{ \"LastName\" : \"Doe\" }, { \"Name\" : \"John\" }] } }, { \"$match\" : { \"Address\" : /^Drive/s } }, { \"$project\" : { \"Age\" : \"$Age\", \"Address\" : \"$Address\", \"Name\" : \"$Name\", \"_id\" : 0 } }])")]
+        public void Query_Syntax_11()
```

Comment:
Try eliminating the repeating prefix Query, and think about more meaningful names for test methods if possible.
Also need test cases for edge cases like:

1. Methods invocation `where Foo(something) `
2. `from item in q where Foo(item)`
3. Don't always use `queryable` name, as it matches the MQLGenerator variable.
4. Few queries in single methods
5. Not supported queries like `item.GetHashCode() == 123` and other.